docs/events/winter_jam_2018.md

```diff
@@ -0,0 +1,44 @@
+<p align="center"> 
+<img src="https://cdn.discordapp.com/attachments/259454515823116290/516399623309623317/winter_jam.png">
+</p>
+
```

Comment:
This image should probably be less orange, and a bit bigger lol.

Reply:
This is the image Jamies gave me would need to ask him for any changes :)